src/control.js

```diff
@@ -47,6 +47,7 @@ export var Geocoder = L.Control.extend({
 
     icon.innerHTML = '&nbsp;';
     icon.type = 'button';
+    icon.setAttribute('aria-label', 'Initiate a new search');
```

Comment:
We should probably use an option for this, like we do for `placeholder` and `errorMessage`, to make it possible to change the text, for example to other languages. Like `this.options.iconLabel` or similar.